2021-05-20 03:42:32,511:DEBUG:certbot._internal.main:certbot version: 1.15.0
2021-05-20 03:42:32,512:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2021-05-20 03:42:32,512:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/usr/share/nginx/html/letsencrypt', '--email', 'robotixclub@nitrr.ac.in', '--agree-tos', '--no-eff-email', '-d', 'robotix.nitrr.ac.in']
2021-05-20 03:42:32,512:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2021-05-20 03:42:32,583:DEBUG:certbot._internal.log:Root logging level set at 20
2021-05-20 03:42:32,583:INFO:certbot._internal.log:Saving debug log to /var/log/letsencrypt/letsencrypt.log
2021-05-20 03:42:32,585:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2021-05-20 03:42:32,593:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Place files in webroot directory
Interfaces: IAuthenticator, IPlugin
Entry point: webroot = certbot._internal.plugins.webroot:Authenticator
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7f34269197f0>
Prep: True
2021-05-20 03:42:32,594:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7f34269197f0> and installer None
2021-05-20 03:42:32,594:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2021-05-20 03:42:32,608:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/124178225', new_authzr_uri=None, terms_of_service=None), 4ccf1848dab134df4a9a7b01a72cbb83, Meta(creation_dt=datetime.datetime(2021, 5, 19, 18, 39, 12, tzinfo=<UTC>), creation_host='30d663a1f659', register_to_eff=None))>
2021-05-20 03:42:32,609:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2021-05-20 03:42:32,615:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2021-05-20 03:42:33,607:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 658
2021-05-20 03:42:33,608:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 20 May 2021 03:42:33 GMT
Content-Type: application/json
Content-Length: 658
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "Uyqicy8gqAQ": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.2-November-15-2017.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert"
}
2021-05-20 03:42:33,609:DEBUG:certbot.display.util:Notifying user: Requesting a certificate for robotix.nitrr.ac.in
2021-05-20 03:42:33,827:DEBUG:certbot.crypto_util:Generating RSA key (2048 bits): /etc/letsencrypt/keys/0026_key-certbot.pem
2021-05-20 03:42:33,834:DEBUG:certbot.crypto_util:Creating CSR: /etc/letsencrypt/csr/0026_csr-certbot.pem
2021-05-20 03:42:33,834:DEBUG:acme.client:Requesting fresh nonce
2021-05-20 03:42:33,835:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2021-05-20 03:42:34,085:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2021-05-20 03:42:34,086:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 20 May 2021 03:42:33 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 0104fmd89b2iO1-N-8IAS8Ta8Cr9_-Ks5e0eF-0Op7eosHU
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2021-05-20 03:42:34,088:DEBUG:acme.client:Storing nonce: 0104fmd89b2iO1-N-8IAS8Ta8Cr9_-Ks5e0eF-0Op7eosHU
2021-05-20 03:42:34,088:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "robotix.nitrr.ac.in"\n    }\n  ]\n}'
2021-05-20 03:42:34,090:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTI0MTc4MjI1IiwgIm5vbmNlIjogIjAxMDRmbWQ4OWIyaU8xLU4tOElBUzhUYThDcjlfLUtzNWUwZUYtME9wN2Vvc0hVIiwgInVybCI6ICJodHRwczovL2FjbWUtdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9uZXctb3JkZXIifQ",
  "signature": "CBoCokHV4jdnJAM4Zzf0ebfN6kPuLsSadTnhoeXYdBPdmP6RoVE4QqZbHoMRG14pB_0LGgCVHS7r1kFAPQH8P9x4fhpbA3_TxXKgRlbbfc_-FY-DJ2UFYZjNXt-a3XK1XRX7mhBUuMsB4fnA9brMt5ZjWgTsLK5_U9-z8D9Vm5WQ4jV2oBxZ0Ve0AjShqj-3B5NahvQpoC7hpEin0ofpOfsxj4xZQTGWeGn0Pwm7CVVmIo0pGX-u0TDxRp1L2n3l3HiJN-Gq4p3q0CLG1Z6djx9jTk-d1Um2MkVexJDp2GheG5JqCGZmBQF3Jb_IlMsE5V7Mbl2BfObaP8BnfbdpuQ",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogInJvYm90aXgubml0cnIuYWMuaW4iCiAgICB9CiAgXQp9"
}
2021-05-20 03:42:34,355:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 341
2021-05-20 03:42:34,355:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Thu, 20 May 2021 03:42:34 GMT
Content-Type: application/json
Content-Length: 341
Connection: keep-alive
Boulder-Requester: 124178225
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/124178225/9821150361
Replay-Nonce: 0103qZpT_MSrUqV35vPr6G6kREQ8f5xwlb669xmVf86l8Rw
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2021-05-27T03:42:34Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "robotix.nitrr.ac.in"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz-v3/13275168814"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/124178225/9821150361"
}
2021-05-20 03:42:34,356:DEBUG:acme.client:Storing nonce: 0103qZpT_MSrUqV35vPr6G6kREQ8f5xwlb669xmVf86l8Rw
2021-05-20 03:42:34,356:DEBUG:acme.client:JWS payload:
b''
2021-05-20 03:42:34,357:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/13275168814:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTI0MTc4MjI1IiwgIm5vbmNlIjogIjAxMDNxWnBUX01TclVxVjM1dlByNkc2a1JFUThmNXh3bGI2Njl4bVZmODZsOFJ3IiwgInVybCI6ICJodHRwczovL2FjbWUtdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9hdXRoei12My8xMzI3NTE2ODgxNCJ9",
  "signature": "REqS-H394Lw4fXdWr0vPtyjVmCRGqL1LvNvlE0TWh8Ncl3o1CZTnnEZ0OXSOqft00gaCncCk7lGVJYVHK-2Vfuhb0QtDDb6rZ0G_KTw5nPugd_-3KSzk3hWAYRh6LyCrPh3PXncG3q_jyQShrP0ksOR4h2aLQbn6hRpbbljWIqpcI0O_WOLQp4uXdBOsWm7KPeipRh9TT5-L96Ia9gxOgy0YgsPCxSbtXhlQbFYAt3AWy5_zaRTWFn11uc49oJtEq7fvRLx2DUeS1F8_k1tPjCuYJz8d55eu1NI2F89H2npho9VhnY5HXqUNcy_lVfdDXJifrY8UB1hI1D_3npQJkQ",
  "payload": ""
}
2021-05-20 03:42:34,609:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/13275168814 HTTP/1.1" 200 800
2021-05-20 03:42:34,610:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 20 May 2021 03:42:34 GMT
Content-Type: application/json
Content-Length: 800
Connection: keep-alive
Boulder-Requester: 124178225
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 0104WkIj2hgB_5TE2sqnoKJIUE2AzsSd88rUMRifSuhOQa4
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "robotix.nitrr.ac.in"
  },
  "status": "pending",
  "expires": "2021-05-27T03:42:34Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275168814/zbb2XQ",
      "token": "0iBxe0zv3yhCPpsjob4nEKbBR_g36yMFbbuJ9vCwYqA"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275168814/lONoCQ",
      "token": "0iBxe0zv3yhCPpsjob4nEKbBR_g36yMFbbuJ9vCwYqA"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275168814/QzbWcg",
      "token": "0iBxe0zv3yhCPpsjob4nEKbBR_g36yMFbbuJ9vCwYqA"
    }
  ]
}
2021-05-20 03:42:34,610:DEBUG:acme.client:Storing nonce: 0104WkIj2hgB_5TE2sqnoKJIUE2AzsSd88rUMRifSuhOQa4
2021-05-20 03:42:34,610:INFO:certbot._internal.auth_handler:Performing the following challenges:
2021-05-20 03:42:34,611:INFO:certbot._internal.auth_handler:http-01 challenge for robotix.nitrr.ac.in
2021-05-20 03:42:34,612:INFO:certbot._internal.plugins.webroot:Using the webroot path /usr/share/nginx/html/letsencrypt for all unmatched domains.
2021-05-20 03:42:34,612:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge
2021-05-20 03:42:34,614:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/0iBxe0zv3yhCPpsjob4nEKbBR_g36yMFbbuJ9vCwYqA
2021-05-20 03:42:34,614:INFO:certbot._internal.auth_handler:Waiting for verification...
2021-05-20 03:42:34,617:DEBUG:acme.client:JWS payload:
b'{}'
2021-05-20 03:42:34,618:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275168814/zbb2XQ:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTI0MTc4MjI1IiwgIm5vbmNlIjogIjAxMDRXa0lqMmhnQl81VEUyc3Fub0tKSVVFMkF6c1NkODhyVU1SaWZTdWhPUWE0IiwgInVybCI6ICJodHRwczovL2FjbWUtdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9jaGFsbC12My8xMzI3NTE2ODgxNC96YmIyWFEifQ",
  "signature": "xUmdaQd3W7WjmHEr8rnoTbFT4nauUszf2r8p_zAr0AMuVo9_lmEiY5YDj0ux5O9U_wz7jI3Z5X22YN1gOzxmsOK3G3HrYI34HMezB6O24HCmTKbphE3LBgjiV51ownIhk7YjEohKHdRDysRrR5pXP8RUUM2l_gpNhQQKMBN0roXVqEAskc8L2vjrT4-wvKCxKAlwQ3p4Jl6POc2G6oDl3pF9iokuFDB1cqh4Gp-RSeNPYZkGI870jQWLNRTPJNVhvi-4nnjHLWJ9wcL5cCt1EFrSjDJTTYpoJJhIX4BE_pD4lcb7kInj_9EqbiF7IhIFJgKJxVos8tGCs4zpTkmHFw",
  "payload": "e30"
}
2021-05-20 03:42:34,869:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall-v3/13275168814/zbb2XQ HTTP/1.1" 200 186
2021-05-20 03:42:34,870:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 20 May 2021 03:42:34 GMT
Content-Type: application/json
Content-Length: 186
Connection: keep-alive
Boulder-Requester: 124178225
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz-v3/13275168814>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275168814/zbb2XQ
Replay-Nonce: 0104b0raBepsHWQtAwKYpGI8lhLrjHC4lch6YArQJ6VK-fk
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "status": "pending",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275168814/zbb2XQ",
  "token": "0iBxe0zv3yhCPpsjob4nEKbBR_g36yMFbbuJ9vCwYqA"
}
2021-05-20 03:42:34,870:DEBUG:acme.client:Storing nonce: 0104b0raBepsHWQtAwKYpGI8lhLrjHC4lch6YArQJ6VK-fk
2021-05-20 03:42:35,872:DEBUG:acme.client:JWS payload:
b''
2021-05-20 03:42:35,873:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/13275168814:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTI0MTc4MjI1IiwgIm5vbmNlIjogIjAxMDRiMHJhQmVwc0hXUXRBd0tZcEdJOGxoTHJqSEM0bGNoNllBclFKNlZLLWZrIiwgInVybCI6ICJodHRwczovL2FjbWUtdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9hdXRoei12My8xMzI3NTE2ODgxNCJ9",
  "signature": "HDfzS4ykEVj16EiZRWJ37fQNOuw_AfeBT1LlU-xOU8SNtsAm_XGF8hlaCqCRVWZM1YGd7ibrL1pnok_5Ty9K6L9QTKYTzl-ivc24DGD8Tj-d6IKp4WkTXhcKDDkmDdAbTrzk3U20Ojk7ToenZ0ZvhTVN0GJnuAH0aN0w1ZRm8pG54h4gKe9vpEbdrc-sGKTqC-_AaxsE4nEQiFeMJMTHNMjGeVOW6Wg0xhgqJFN0ikippoc2pAsH3TZJpc2xHLjjc8oyzjfvJor44v8H21PcCe0d1jiG8f6I0lGKhMfpJ4PGKgUuWAuzRD1Ss_nvfThEZebCfrAbl32IFwqnc7DV8w",
  "payload": ""
}
2021-05-20 03:42:36,123:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/13275168814 HTTP/1.1" 200 1308
2021-05-20 03:42:36,124:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 20 May 2021 03:42:35 GMT
Content-Type: application/json
Content-Length: 1308
Connection: keep-alive
Boulder-Requester: 124178225
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 0104B8f-Fdduvq0945w2LGCzxUkW_xQpOs8UB9bGBcQXq5g
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "robotix.nitrr.ac.in"
  },
  "status": "invalid",
  "expires": "2021-05-27T03:42:34Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "invalid",
      "error": {
        "type": "urn:ietf:params:acme:error:unauthorized",
        "detail": "Invalid response from http://robotix.nitrr.ac.in/.well-known/acme-challenge/0iBxe0zv3yhCPpsjob4nEKbBR_g36yMFbbuJ9vCwYqA [3.108.0.154]: \"\u003chtml\u003e\\r\\n\u003chead\u003e\u003ctitle\u003e404 Not Found\u003c/title\u003e\u003c/head\u003e\\r\\n\u003cbody\u003e\\r\\n\u003ccenter\u003e\u003ch1\u003e404 Not Found\u003c/h1\u003e\u003c/center\u003e\\r\\n\u003chr\u003e\u003ccenter\u003enginx/1.20.0\u003c/ce\"",
        "status": 403
      },
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275168814/zbb2XQ",
      "token": "0iBxe0zv3yhCPpsjob4nEKbBR_g36yMFbbuJ9vCwYqA",
      "validationRecord": [
        {
          "url": "http://robotix.nitrr.ac.in/.well-known/acme-challenge/0iBxe0zv3yhCPpsjob4nEKbBR_g36yMFbbuJ9vCwYqA",
          "hostname": "robotix.nitrr.ac.in",
          "port": "80",
          "addressesResolved": [
            "3.108.0.154"
          ],
          "addressUsed": "3.108.0.154"
        }
      ],
      "validated": "2021-05-20T03:42:34Z"
    }
  ]
}
2021-05-20 03:42:36,124:DEBUG:acme.client:Storing nonce: 0104B8f-Fdduvq0945w2LGCzxUkW_xQpOs8UB9bGBcQXq5g
2021-05-20 03:42:36,124:WARNING:certbot._internal.auth_handler:Challenge failed for domain robotix.nitrr.ac.in
2021-05-20 03:42:36,125:INFO:certbot._internal.auth_handler:http-01 challenge for robotix.nitrr.ac.in
2021-05-20 03:42:36,126:DEBUG:certbot._internal.reporter:Reporting to user: The following errors were reported by the server:

Domain: robotix.nitrr.ac.in
Type:   unauthorized
Detail: Invalid response from http://robotix.nitrr.ac.in/.well-known/acme-challenge/0iBxe0zv3yhCPpsjob4nEKbBR_g36yMFbbuJ9vCwYqA [3.108.0.154]: "<html>\r\n<head><title>404 Not Found</title></head>\r\n<body>\r\n<center><h1>404 Not Found</h1></center>\r\n<hr><center>nginx/1.20.0</ce"

To fix these errors, please make sure that your domain name was entered correctly and the DNS A/AAAA record(s) for that domain contain(s) the right IP address.
2021-05-20 03:42:36,126:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 91, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 179, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2021-05-20 03:42:36,126:DEBUG:certbot._internal.error_handler:Calling registered functions
2021-05-20 03:42:36,126:INFO:certbot._internal.auth_handler:Cleaning up challenges
2021-05-20 03:42:36,127:DEBUG:certbot._internal.plugins.webroot:Removing /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/0iBxe0zv3yhCPpsjob4nEKbBR_g36yMFbbuJ9vCwYqA
2021-05-20 03:42:36,127:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2021-05-20 03:42:36,127:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
  File "/opt/certbot/src/certbot/certbot/main.py", line 15, in main
    return internal_main.main(cli_args)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1435, in main
    return config.func(config, plugins)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1304, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 140, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 444, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 374, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 424, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 91, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 179, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2021-05-20 03:42:36,128:ERROR:certbot._internal.log:Some challenges have failed.
